report/web.py

```diff
@@ -166,12 +166,33 @@ def get_results(benchmark) -> tuple[list[evaluator.Result], list[str]]:
   return results, targets
 
 
+def prepare_prompt_for_html_text(raw_prompt_content: str) -> str:
+  """Converts a raw prompt file into presentable HTML text."""
+  try:
+    structured_prompt = json.loads(raw_prompt_content)
+    if isinstance(structured_prompt, list) and len(structured_prompt) > 0:
+      html_presentable_content = ''
+      for elem in structured_prompt:
+        if isinstance(elem, dict) and 'content' in elem:
+          html_presentable_content += '\n%s' % (elem['content'])
+      return html_presentable_content
+  except json.decoder.JSONDecodeError:
```

Comment:
Could you please log an error to help with debugging?

Reply:
done